grid_map_cv/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.5.1)
 project(grid_map_cv)
 
-set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
 add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
set(CMAKE_CXX_STANDARD 17)

?

Reply:
The idea here is that I assume every system this is building on now is >= 11 so on Ubuntu 20.04 the default will be 14, and on 22.04 it will be 17, there's no reason to hold it back at 11.